packages/imodel-react-hooks/src/FeatureOverrides/useFeatureOverrides.test.tsx

```diff
@@ -0,0 +1,142 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
```

Comment:
the internal copyright is still here for all the files, might want to delete that

Reply:
totally missed that, thank you, fixing